doc/src/mailserver.md

```diff
@@ -97,15 +96,16 @@ There must always be exactly one domain with the primary option set.
 This sets up [autoconfiguration] for mail clients that wish to use *test.fcio.net*.
 Autoconfiguration is disabled for *test2.fcio.net* in the example.
 
-Run {command}`sudo fc-manage -b` to have everything configured on the system.
+Run {command}`sudo fc-manage switch` to have everything configured on the system.
 
-Afterwards, a generated file {file}`/etc/local/mail/dns.zone` contains all
-necessary DNS settings for your mail server. Insert the records found in this
-file into the appropriate DNS zones and don't forget to check reverses.
+After running the above command, a newly-generated file {file}`/etc/local/mail/dns.zone`
+will contain all necessary DNS records for your mail server.
+Insert the records contained within the file into the appropriate DNS zones and
+don't forget to check PTR records for reverse DNS lookups.
 
 ## How do I create users?
 
-Edit {file}`/etc/local/mail/users.json` to add user accounts. Example:
+Edit the file {file}`/etc/local/mail/users.json` to add user accounts. Example:
```

Comment:
Hm … the users can be defined via nix, too, can't they? Can we document both?

Reply:
Both are documented now